cypress/fixtures/other/generic-product.json

```diff
@@ -0,0 +1,24 @@
+{
+  "serviceName": "CY Credit Note Test",
+  "bookingReferenceId": "CY ABC123",
+  "tripId": "981ba1a0-f9cc-48f0-9644-1343d6e70929",
```

Comment:
Is this ID always exist in our system? Whats happens with the test in a different environment? Ex. Staging?

Reply:
It exists on develop environment. For test case which depends to this data, it's only important that the `tripId` is invalid/inactive, so it will work for other environments too.

Reply:
We need to figure out a different solution for this. All tests should be executable in all environments.